teams/teams-ps/teams/Add-TeamChannelUser.md

```diff
@@ -10,9 +10,9 @@ schema: 2.0.0
 ## SYNOPSIS
 Adds an owner or member to the private channel.
 
-Note: the command will return immediately, but the Teams application will not reflect the update immediately, please refresh the members page to see the update.
+Note: The command will return immediately, but the Teams application will not reflect the update immediately. Please refresh the members page to see the update.
 
-To turn an existing Member into an Owner,  first Add-TeamChannelUser -User foo to add them to the members list, then Add-TeamChannelUser -User foo -Role Owner to add them to owner list.
+Note: Technical limitations of private channels apply. To add a user as a member to a channel, they need to first be a member of the team. To make a user an owner of a channel, they need to first be a member of the channel. In some cases, this might require to call `Add-TeamChannelUser` twice.
```

Comment:
```suggestion
Note: Technical limitations of private channels apply. To add a user as a member to a channel, they need to first be a member of the team. To make a user an owner of a channel, they need to first be a member of the channel. In some cases, this might require you to call `Add-TeamChannelUser` twice.
```